core/resources/scene/config.yaml

```diff
@@ -177,7 +183,10 @@ layers:
         data: {source: osm, layer: roads}
         filter: { name: true, aeroway: false, tunnel: false, railway: false, not: { kind: rail }, $zoom: { min: 10 } }
         draw:
-            text: {}
+            text: 
+                font:
+                    typeface: Regular 15 Roboto
```

Comment:
With respect to implementation point of view on cpp:
This gets really hard and ugly to parse typeface string into "face", "weight", "size" and "name" for further processing. And given these parameters could be optional, parsing code can get really ugly and convoluted.

But if we have yaml key value pairs for these parameters, it will be trivial implementation for both cpp and js side.